### PR TITLE
Cleanup cleanup-doc

### PR DESCRIPTION
**Pull request by cmsj**

### CODE REVIEW

**Comment by cmsj** [comment]
This one needs a closer look before merging, there are some odd looking changes.

File: extensions/doc/makeLuaDocs.lua [hunk 1]
```diff
@@ -195,7 +195,8 @@ print("++ Parsing manual...")
 
 -- Get manual version first -- first pass we ignore keys that aren't numbers
 
-    local a, b, k = 1, 0, ""
+    local a = 1
+    local b, k
```
Comment (cmsj) [inline extensions/doc/makeLuaDocs.lua]: This seems potentially concerning.

File: extensions/doc/makeLuaDocs.lua [hunk 2]
```diff
@@ -230,8 +231,8 @@ print("++ Parsing manual...")
 -- ignorable, as they're likely links within the manual itself to lists already included in a manual section,
 -- but post them just in case I need to look for them later if the format changes.
 
-    local a, b, k = 1, 0, ""
-    local text, posTable = nil, nil
+    a = 1
```
Comment (cmsj) [inline extensions/doc/makeLuaDocs.lua]: as does this

File: extensions/doc/makeLuaDocs.lua (hunk 1 above)
Comment (latenitefilms) [reply extensions/doc/makeLuaDocs.lua]: `b` and `k` are redefined straight away [here](https://github.com/cmsj/hammerspoon/blob/d5c32b2799b534ae355cfeb3615831f052cc1f42/extensions/doc/makeLuaDocs.lua#L203):

```lua
a, b, k = LuaManual:find("[\r\n][^\r\n]*<[aA][^\r\n]*%s+[nN][aA][mM][eE]%s*=%s*\"([%d%.]+)\"[^\r\n]*>", a)
```

Therefore, their original value is never used.

File: extensions/doc/makeLuaDocs.lua (hunk 2 above)
Comment (latenitefilms) [reply extensions/doc/makeLuaDocs.lua]: `b` and `k` are already defined as `local` [earlier in the code](https://github.com/cmsj/hammerspoon/blob/d5c32b2799b534ae355cfeb3615831f052cc1f42/extensions/doc/makeLuaDocs.lua#L199). Their initial values are also overwritten [here](https://github.com/cmsj/hammerspoon/blob/d5c32b2799b534ae355cfeb3615831f052cc1f42/extensions/doc/makeLuaDocs.lua#L203).

`text` and `posTable` are also already defined as `local` [earlier in the code](https://github.com/cmsj/hammerspoon/blob/d5c32b2799b534ae355cfeb3615831f052cc1f42/extensions/doc/makeLuaDocs.lua#L200).

**Pull request merged**